.github/workflows/main.yml

```diff
@@ -74,21 +74,45 @@ jobs:
           tag_name: ${{ steps.git_tag.outputs.tag }}
           release_name: Release ${{ steps.git_tag.outputs.tag }}
           body: ${{ steps.git_tag.outputs.changes }}
-      - if: github.ref == 'refs/heads/master'
-        name: Prepare chart for publishing
-        id: chart_prepare
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: github-ci
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_ICON: https://github.com/redkubes.png?size=48
+          SLACK_TITLE: CI run ${{ job.status }}
+          SLACK_USERNAME: RedKubesBot
+        if: always() # failure
+
+  chart-release:
+    if: "contains(github.event.head_commit.message, 'chore(release)')"
+    needs: build-test-push-release
+    runs-on: ubuntu-latest
+    container:
+      image: otomi/tools:v1.4.19
+      options: --user 0
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Check for changes
+        id: prereq
         run: |
           # Release new chart only if its version changes
-          [ ! $(git show ${{ github.sha }} chart/otomi/Chart.yaml | grep '^+version:.*$') ] && echo "No chart changes detected. Skipping." && exit
-          echo ::set-output name=has_chart_changes::true
+          helm repo add otomi https://otomi.io/otomi-core
+          helm repo update
+          this_version=v$(cat chart/otomi/Chart.yaml | yq r - 'version')
+          chart_version=$(helm search repo otomi | sed -n 2p | awk '{print $2}')
```

Comment:
Can we read chart version in more explicit way
```
helm search repo otomi -o json | jq  -r '.[-1].version'
```
Please be aware that in both cases we assume that there is only one chart in chart repo

Reply:
unfortunately not, as that would pull the oldest version, not the latest...can you give me a selector that gets the latest version then?

(I tried with `helm search repo jetstack/cert-manager -o json | jq  -r '.[-1].version'`)

Reply:
solved with latest commit